snapshot/src/main/scala/com/evolutiongaming/kafka/journal/SnapshotSelectionCriteria.scala

```diff
@@ -0,0 +1,26 @@
+package com.evolutiongaming.kafka.journal
+
+import java.time.Instant
+
+/** Type safer version of `akka.persistence.SnapshotSelectionCriteria`.
+  * 
+  * The important consideration when using this class is to always use named
+  * parameters to consruct the instances of the class. Otherwise it is too
+  * easy to confuse `maxSeqNr` and `minSeqNr`, especially, given the fact,
+  * they are given in a reverse order in a constructor.
+  */
+final case class SnapshotSelectionCriteria(
```

Comment:
Just to be on a same page: new akka-effect persistence API does not accept the criteria https://github.com/evolution-gaming/akka-effect/blob/262cc7b70a2f7419644d9c4b7644b4b1888f5733/persistence-api/src/main/scala/com/evolutiongaming/akkaeffect/persistence/SnapshotStore.scala#L19

thus in integration always will be used `SnapshotSelectionCriteria.All`, IMO its okay